server/src/test/java/com/angelomelonas/grpcwebchat/ChatServiceTest.java

```diff
@@ -1,4 +1,19 @@
 package com.angelomelonas.grpcwebchat;
 
+import org.junit.Test;
+
 public class ChatServiceTest {
+
```

Comment:
I would suggest you drive your development from this class... add meat to your tests and then implement the code from this.